src/providers/arcgisrest/qgsafsprovider.cpp

```diff
@@ -249,20 +249,21 @@ QgsAfsProvider::QgsAfsProvider( const QString &uri, const ProviderOptions &optio
   {
     const QVariantMap timeInfo = layerData.value( QStringLiteral( "timeInfo" ) ).toMap();
 
-    temporalCapabilities()->setHasTemporalCapabilities( true );
-    temporalCapabilities()->setStartField( timeInfo.value( QStringLiteral( "startTimeField" ) ).toString() );
-    temporalCapabilities()->setEndField( timeInfo.value( QStringLiteral( "endTimeField" ) ).toString() );
-    if ( !temporalCapabilities()->endField().isEmpty() )
-      temporalCapabilities()->setMode( Qgis::VectorDataProviderTemporalMode::StoresFeatureDateTimeStartAndEndInSeparateFields );
-    else if ( !temporalCapabilities()->startField().isEmpty() )
-      temporalCapabilities()->setMode( Qgis::VectorDataProviderTemporalMode::StoresFeatureDateTimeInstantInField );
+    QgsVectorDataProviderTemporalCapabilities *l_temporalCapabilities = temporalCapabilities();
```

Comment:
I've seen this elsewhere but never asked -- what's the "l_" prefix signify? Does it stand for something?

Reply:
l_ = local variable. As a way to disambiguate the virtual method foo() and the local variable storing its value.
But actually I see in src/core/qgsmaplayer.cpp a "const QgsDataProvider *lDataProvider = dataProvider()". Is "lFoo" a clearer convention than "l_foo" ?

Reply:
I think so -- _ isn't used widely in qgis naming.

Reply:
ok, commit ammended to use lFoo convention